theme.go

```diff
@@ -30,6 +30,8 @@ var (
 		TableHeaderStyle:        Style{FgLightCyan},
 		TableSeparatorStyle:     Style{FgGray},
 		SectionStyle:            Style{Bold, FgYellow},
+		ListTextStyle:           Style{FgWhite},
+		ListBulletStyle:         Style{FgWhite},
```

Comment:
```suggestion
		ListBulletStyle:         Style{FgGray},
```
This would probably look better in `FgGray`